hwloc/topology-rsmi.c

```diff
@@ -165,7 +213,9 @@ hwloc_rsmi_discover(struct hwloc_backend *backend, struct hwloc_disc_status *dst
   for (i=0; i<nb; i++) {
     uint64_t bdfid = 0;
     hwloc_obj_t osdev, parent;
-    char buffer[64];
+    char buffer[80];
```

Comment:
This change is unneeded now. Otherwise we're good, thanks a lot.
The other commits with unrelated changes can go in the same PR, I'll merge both at once.

Reply:
Can you rebase on top of latest master? Some tests in the CI require recent commits in master.